main.go

```diff
@@ -47,11 +47,18 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
 
+	operatorNamespace := os.Getenv("OPERATOR_NAMESPACE")
+	if operatorNamespace == "" {
+		log.Info("unable to find operator namespace")
+		os.Exit(1)
+	}
```

Comment:
This actually breaks the PR workflow in GitHub Actions. We have the same code in the Cluster Operator, and it is a good thing that we are now consistent. However, the `manager.yaml` file is missing the following bits to deploy successfully:

https://github.com/rabbitmq/cluster-operator/blob/fec7f9c666097ea2cf9a99d8dfc35e08bfe4432c/config/manager/manager.yaml#L45-L49